third_party/rake-compiler-dock/push_testing_images.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Can we remove this script, and rely on `tools/dockerfile/push_testing_images.sh` to take care of these images?

Reply:
I deliberately did it because this is a folk of `rake-compiler-dock` and resides in third_party directory. I expect that this will be eventually merged in the upstream `rake-compiler-dock`.

Reply:
Even though this is a fork, I don't see the reason for having two manual scripts to run (it only adds overhead).

Reply:
Other than the fact that they're docker files and stored in the same registry, the time to run the two scripts are pretty different. For the rake-compiler-dock, probably I will do for a while only when they need some changes so naturally both docker files and a script to upload can be in the same directory. For other existing docker files, they're quite irrelevant to this. WDTY?

Reply:
I don't see any advantage of having them separate from the rest, though.

Even if these dockerfiles serve different purposes it breaks the usual workflow of running `tools/dockerfile/push_testing_images.sh` after making any changes to dockerfiles, which could take an extra debugging round when making changes.